Dockerfile.base

```diff
@@ -21,10 +20,7 @@
 # - put the tag into the FROM line in Dockerfile (of the main image)
 #   temporarily, rebuild that one, test, etc.
 #
-# TODO(DCOS-42576): set a periodic job to update the FROM line in Dockerfile,
-# so that we pick up updates regularly but in a controlled manner.
-#
-FROM ubuntu:18.04
+FROM ubuntu:18.04@sha256:29934af957c53004d7fb6340139880d23fb1952505a15d69a03af0d1418878cb
```

Comment:
What does this SHA represent?

Reply:
This is the unique content-checksum of a particular build of the image. Printed at the end of `docker pull`.